pkg/alerts/alertsqlite/alerts_sqlite.go

```diff
@@ -137,6 +137,8 @@ func (p Sqlite) verifyAlertExists(alert_id string) (bool, *alertutils.AlertDetai
 		}
 	}
 
+	_ = alert.DecodeQueryParamFromBase64()
+
```

Comment:
I did not check , because the error case would not occur. As if you see before `Decoding` two lines above or so I am doing `Encoding`. So, I felt it would be redundant to check. As it is a sure case. 
But do you think it is better to check the errors?

Reply:
I see. I think it'll still be better to check for errors. Even if it's impossible for the error to occur currently, we may later change this code and potentially that could make the error possible, and then we'd want to make sure we do proper error handling; I think it'll make the code easier to maintain if you add the error handling now